requirements.txt

```diff
@@ -1,4 +1,8 @@
+BeneDict==0.3.2
```

Comment:
Btw, it appears `BeneDict` is not the correct package, `python-benedict` is.
In order to fix an import issue I had to uninstall `BeneDict` and `python-benedict` to un-snarl pip, then reinstall `python-benedict`

I recommend removing the faulty package from your req file:
```
python@51713e846bd0:/app/codecollection/codebundles/aws-service-health$ git diff
diff --git a/requirements.txt b/requirements.txt
index ea22d46..2a583d8 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,3 @@
-BeneDict==0.3.2
```

Reply:
Fair point, I also deleted some other unnecessary packages from requirements